end-to-end-tests/client_util.go

```diff
@@ -0,0 +1,60 @@
+// (c) Cartesi and individual authors (see AUTHORS)
+// SPDX-License-Identifier: Apache-2.0 (see LICENSE)
+
+package endtoendtests
+
+import (
+	"context"
+
+	"github.com/cartesi/rollups-node/pkg/addresses"
+	"github.com/cartesi/rollups-node/pkg/ethutil"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/ethereum/go-ethereum/rpc"
+)
+
+func AddInput(ctx context.Context,
+	blockchainHttpEnpoint string, payload string) (int, error) {
+
+	// Send Input
+	client, err := ethclient.DialContext(ctx, blockchainHttpEnpoint)
+	if err != nil {
+		return 0, err
+	}
+	defer client.Close()
+
+	signer, err := ethutil.NewMnemonicSigner(ctx, client, ethutil.FoundryMnemonic, 0)
+	if err != nil {
+		return 0, err
+	}
+	book := addresses.GetTestBook()
+
+	payloadBytes, _ := hexutil.Decode(payload)
+	inputIndex, err := ethutil.AddInput(ctx, client, book, signer, payloadBytes)
+	if err != nil {
+		return 0, err
+	}
+	return inputIndex, nil
+
+}
+
+func AdvanceTime(ctx context.Context, blockchainHttpEnpoint string, timeInSeconds int) error {
```

Comment:
Just found out that `cartesi-rollups-cli` has another implementation of this, using JSON RCP instead of Go Etherium. We should have only one

Reply:
Should I change the Command to use this one? and move this util functions to `ethutil.go` ?